docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '3'
```

Comment:
is this file in scope of this ticket?

Reply:
@lzach83 why is this file edited/added in this PR? It was my understanding this was part of Graham's PR.